src/Confluent.Kafka/Serialization/IDeserializer.cs

```diff
@@ -26,13 +30,19 @@ public interface IDeserializer<T>
         ///     Deserialize a byte array to an instance of
         ///     type T.
         /// </summary>
+        /// <param name="topic">
+        ///     The topic associated wih the data.
+        /// </param>
         /// <param name="data">
         ///     The serialized representation of an instance
         ///     of type T to deserialize.
         /// </param>
         /// <returns>
         ///     The deserialized value.
         /// </returns>
-        T Deserialize(byte[] data);
+        T Deserialize(string topic, byte[] data);
+
+        /// <include file='../include_docs.xml' path='API/Member[@name="IDeserializer_Configure"]/*' />
```

Comment:
We have quite a bit of API doc duplication, so I want to start doing this in other places as well.